src/brizo/Brizo.ts

```diff
@@ -119,13 +119,24 @@ export class Brizo extends Instantiable {
         jobId?: string,
         output?: Output
     ): Promise<ComputeJob | ComputeJob[]> {
-        const signature = await this.createSignature(consumerAccount, serviceAgreementId)
         const address = consumerAccount.getId()
-        const serviceEndpoint = await this.getEndpointFromAgreement(
-            'compute',
-            serviceAgreementId
+
+        let signatureMessage = address
+        signatureMessage += (jobId && `${jobId}`) || ''
```

Comment:
This would be the same but easier to read:

```suggestion
        signatureMessage += jobId || ''
```

`jobId` can only be passed as string because of typing, so no need to stringify it again.